src/System/Console/Hawk/Args/Parse.hs

```diff
@@ -18,7 +18,7 @@ import           System.Console.Hawk.Context.Dir
 
 
 -- | (record separator, field separator)
-type CommonSeparators = (Separator, Separator)
+type CommonSeparators = (Separator, Separator, String)
```

Comment:
The input file is not a separator; please rename this to something like "CommonOptions". And while we're modifying this, let's make it a record instead of a tuple, to clarify the purpose of each field.

Reply:
Also, if it's a filename, let's use the type synonym FilePath instead of String